docker-compose.yml

```diff
@@ -18,18 +18,22 @@ services:
       - ${VOLUME_ROOT:-.}/volumes/code:/etc/puppetlabs/code/
       - ${VOLUME_ROOT:-.}/volumes/puppet:/etc/puppetlabs/puppet/
       - ${VOLUME_ROOT:-.}/volumes/serverdata:/opt/puppetlabs/server/data/puppetserver/
+    networks:
+      default:
+        aliases:
+         - puppet.${AZURE_DOMAIN}
```

Comment:
Should we put a default value in here for this so we don't get a WARNING about an undefined ENV variable everywhere except when it runs in Azure?

Reply:
I'm good with a WARNING if we can't; it's worth it to get rid of multiple compose files to me.

Reply:
Ah yeah, good catch... let me do that real quick.